aiohttp_json_rpc/client.py

```diff
@@ -156,3 +156,57 @@ def add_methods(self, *methods):
             del self._handler[topic]
 
         return await self.call('unsubscribe', params=topic, timeout=timeout)
+
+
+class JsonRpcMagicClientMethod:
```

Comment:
Hmmmm i like the idea but i dont like the name "magic". Maybe something like simple- or lazy- ?

Reply:
I didn't know how to call it, the only thing was that it uses `"__magic__"` methods, so I called it this way to not waste time for naming :)
I'll call it whatever we agree to do. I think, `JsonRpcLazyClientMethod` would fit. I also think that we might need to prepend it with underscore to indicate that it's not intended for end-users.

Reply:
How about ``JsonRpcTransparentClientMethod`` ? :)